qiskit/transpiler/target.py

```diff
@@ -695,25 +710,31 @@ def get_calibration(
 
         Args:
             operation_name: The name of the operation for the instruction.
-            qargs: The tuple of qubit indices for the instruction.
+            qargs: The qubit indices for the instruction.
             args: Parameter values to build schedule if any.
+            operation_params: The parameters for the Instruction.
```

Comment:
Same comment as in `has_calibration` regarding the order of params for multi-param gates